lib/app/widgets/category_text_widget.dart

```diff
@@ -0,0 +1,19 @@
+import '../theme/global_imports.dart';
+
+class CategoryTextWidget extends StatelessWidget {
+  const CategoryTextWidget({super.key, required this.categoryText});
+
+  final String categoryText;
+
+  @override
+  Widget build(BuildContext context) {
+    return Align(
+      alignment: Alignment.centerLeft,
+      child: Text(
+        categoryText,
+        textAlign: TextAlign.left,
+        style: context.tht.displayMedium,
+      ),
+    );
+  }
+}
```

Comment:
This shouldn't be in the global widgets folder because it concerns a specific page